addons/im_livechat/models/mail_channel.py

```diff
@@ -169,6 +172,34 @@ def _close_livechat_session(self, **kwargs):
             self.message_post(author_id=self.env.ref('base.partner_root').id,
                               body=self._get_visitor_leave_message(**kwargs), message_type='comment', subtype_xmlid='mail.mt_comment')
 
+    def _process_chatbot_next_step(self, step_id):
```

Comment:
"next" is a bit strange.

Simply "_chatbot_process_step" would be enough?

(Also, can you move it into the "Chatbot" section I added in the file?"
Idea is to group our new method and sort them to keep this clean).